<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dependent</class>
 <widget class="QDialog" name="dependent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>303</width>
    <height>196</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>303</width>
    <height>196</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>160</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QSpinBox" name="spinBox_main_h"/>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox_main_m"/>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox_main_s"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QRadioButton" name="radioUp">
        <property name="text">
         <string>countUp</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioDown">
        <property name="text">
         <string>countDown</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="pushStart">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushStop">
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushSet">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>140</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Set</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Main Timer</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Dependent Timer</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkReset">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>170</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Do Not Reset</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>140</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QSpinBox" name="spinBox_dep_h"/>
    </item>
    <item>
     <widget class="QSpinBox" name="spinBox_dep_m"/>
    </item>
    <item>
     <widget class="QSpinBox" name="spinBox_dep_s"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushAbout">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>30</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushStop</sender>
   <signal>clicked()</signal>
   <receiver>dependent</receiver>
   <slot>stop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>237</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>stop()</slot>
 </slots>
</ui>
